cwa-risk-assessment.md

```diff
@@ -1,9 +1,14 @@
 # How does the Corona-Warn-App identify an increased risk?
 
----
-**Later calculation improvements**
+Please note that this document is _not_ up-to-date anymore. It reflects the original risk assessment for the Corona-Warn-App before version 1.5 (released in October 2020) and is kept here for historical reference only. An up-to-date description of risk assessment and risk calculation in the Corona-Warn-App is available in the [Solution Architecture > Mobile Application](./solution_architecture.md#mobile-applications) document section.
+
+<br /><br />
+
+## DEPRECATION NOTICE
```

Comment:
Should there be an "IMPORTANT!" here, to match the German translation?

Reply:
@Ein-Tim see https://github.com/corona-warn-app/cwa-documentation/pull/829/commits/e7997365a93c8f241a9eb689e5421ad3217b908c